package.json

```diff
@@ -27,6 +27,7 @@
     "build": "tsc -p tsconfig.json",
     "lint": "tslint -p tsconfig.json -c tslint.json",
     "test": "jest",
+    "test:watch": "jest --watch",
```

Comment:
Just adding this as a convenience for myself to allow running tests on each file change.